UltiSnips/javascript_jasmine.snippets

```diff
@@ -16,169 +12,58 @@ it('${1:description}', function() {
 });
 endsnippet
 
-snippet bef "before each (js)" b
-beforeEach(function() {
-	$0
-});
-endsnippet
-
-snippet aft "after each (js)" b
-afterEach(function() {
-	$0
-});
-endsnippet
-
-snippet any "any (js)" b
-jasmine.any($1)
-endsnippet
-
-snippet ru "runs (js)" b
-runs(function() {
-	$0
-});
-endsnippet
-
-snippet wa "waits (js)" b
-waits($1);
-endsnippet
-
-snippet ex "expect (js)" b
```

Comment:
I disgree `ee` is far too short a trigger for this.

Other BDD testing framework snippets I've looked at don't have many snippets like these, and the few that do have triggers like the one used here, or longer.